Bump version and add changelog entry (#631)
NordicSemiconductor/pc-nrfconnect-launcher: Changelog.md

## 3.9.3 - 2022-01-04
### Fixed

> - Selecting device would always prompt firmware upgrade even if the firmware on 
>   the device was the same as the bundled firmware.

Reviewer: ```suggestion
- Selecting device no longer prompts firmware upgrade when the firmware on 
  the device was the same as the bundled firmware.
```
